crates/sovereign-sdk/rollup-interface/src/state_machine/soft_confirmation.rs

```diff
@@ -109,9 +122,34 @@ impl<'txs, Tx: BorshSerialize> UnsignedSoftConfirmation<'txs, Tx> {
     //   when removing this fn
     // FIXME: ^
     pub fn pre_fork1_hash<D: Digest>(&self) -> Output<D> {
-        let raw = borsh::to_vec(&self).unwrap();
+        let old = OldFormat {
+            l2_height: self.l2_height,
+            da_slot_height: self.da_slot_height,
+            da_slot_hash: self.da_slot_hash,
+            da_slot_txs_commitment: self.da_slot_txs_commitment,
+            blobs: self.blobs,
+            deposit_data: self.deposit_data.clone(),
+            l1_fee_rate: self.l1_fee_rate,
+            timestamp: self.timestamp,
+        };
+
+        let raw = borsh::to_vec(&old).unwrap();
         D::digest(raw.as_slice())
     }
+
+    /// TODO: Documentation
+    pub fn get_old_format(&self) -> OldFormat<'txs> {
```

Comment:
This can be in an 
```
impl Into<UnsignedSoftConfirmation> for UnsignedSoftConfirmationV1 
```

Reply:
These changes are cherry-picked from #1514 
I believe I should wait for him to change these parts

Reply:
Commented there on this.